frontends/election-manager/src/screens/manual_data_import_precinct_screen.test.tsx

```diff
@@ -36,8 +42,13 @@ test('displays error screen for invalid precinct', () => {
       electionDefinition: electionWithMsEitherNeitherDefinition,
     }
   );
-  getByText('Error: Could not find precinct 12345.');
-  getByText('Back to Index');
+  await screen.findByText('Error: Could not find precinct 12345.');
+  screen.getByText('Back to Index');
+  // There's no change in the invalid precinct case after the writeInQuery
+  // fires, so we just have to wait to avoid a test warning.
+  await act(async () => {
+    await sleep(1);
+  });
```

Comment:
In this case, `advancePromises` didn't do the trick. I actually had to sleep. Could probably do the same by using fake timers and using `advanceTimersAndPromises`